content/vault/v1.21.x/content/docs/secrets/terraform.mdx

```diff
@@ -88,6 +109,13 @@ Teams, Legacy Team Tokens and Users. Each token type has distinct access levels
 and generation workflows. A given Vault role can manage any one of the 
 supported types at a time, however there are important differences to be aware of.
 
+~> **IMPORTANT:** When selecting an anchor credential for the Terraform Secrets 
+Engine, choose a token whose scope matches your intended use case. For example, a 
+user token can only manage that user’s own resources, while a team token can manage 
+team-level resources or other teams’ tokens if the appropriate permissions are granted. 
+Using a token that is too narrowly scoped may prevent Vault from issuing tokens for your 
+intended workflows.
```

Comment:
```suggestion
When you select an anchor credential for the Terraform plugin, always choose a
token with a scope that matches your intended use case.

User tokens should only have permission to manage the resources owned by that
user. Team tokens should only have permission to manage team-level resources
owned by that team. Only create tokens with cross-team permissions when a more
narrowly-defined scope prevents Vault from issuing appropriate tokens or
interferes with your intended workflows.
```
Style correction: write in active voice, provide explicit guidance whenever possible, avoid unnecessary callouts (warnings, tips, etc.)